### PR TITLE
Fix local install of console and update to latest version

### PR DESCRIPTION
**Pull request by chainlink**
To run console locally run the following: 

```
helm install pachyderm etc/helm/pachyderm --set console.enabled=true -f entkeyhelm.yaml -f etc/helm/examples/local-dev-values.yaml
```

Note: running with local-values.yaml does not work due to a bug with the port number assignment when not using node port

Questions 
- [x] Does `REACT_APP_RUNTIME_SUBSCRIPTIONS_PREFIX` now need to be set in all environments?
- [x] Is the current setup for  `REACT_APP_RUNTIME_ISSUER_URI`  best practice? (With trailing slash)

### CODE REVIEW

File: etc/helm/examples/local-dev-values.yaml [hunk 1]
```diff
@@ -50,7 +50,7 @@ pachd:
 console:
   enabled: false
   config:
-    issuerURI: "http://pachd:1658/"
+    issuerURI: "http://pachd:1658"
```
Comment (acohen4) [inline etc/helm/examples/local-dev-values.yaml:53]: is this trailing slash removed on purpose? I think `console.issuerURI `must equal `pachd.oauthIssuer`

File: etc/helm/pachyderm/values.yaml [hunk 2]
```diff
@@ -53,7 +53,7 @@ console:
     #  cpu: "1"
     #  memory: "2G"
   config:
-    issuerURI: ""
+    issuerURI: "" # Cannot have trailing slash
```
Comment (acohen4) [inline etc/helm/pachyderm/values.yaml:56]: what's the reasoning here? Is this discussed in some convo I can reference?

**Comment by cpepin** [comment]
@chainlink @acohen4 
>  Does REACT_APP_RUNTIME_SUBSCRIPTIONS_PREFIX now need to be set in all environments?

Nope, and I'm guessing this will only need to be explicitly set for local deployments. The default value is `/graphql` which works great when the host name isn't localhost `http://localhost/graphql` vs. `https://hub.pachyderm.com/graphql`.

 > Is the current setup for REACT_APP_RUNTIME_ISSUER_URI best practice? (With trailing slash)

I don't know what is necessarily best practice here, I think it's mostly a side-effect of the implementation. We just take the `ISSUER_URI`, append the auth path, and redirect the client browser. It's a naive implementation, and is just using template strings in the client code (i.e. `${issuerUri}${authPath}`). We can presumable make this work with or without a trailing slash, and to me that seems to be the correct approach. I can work on a PR for that.

File: etc/helm/examples/local-dev-values.yaml (hunk 1 above)
Comment (acohen4) [reply etc/helm/examples/local-dev-values.yaml:53]: based on @cpepin 's  comments, I would think we should remove the trailing slash from `pachd.oauthIssuer`

**Review by acohen4 — APPROVED** [review]
LGTM

**Comment by chainlink** [comment]
I'm going to merge this one and then remove when haberdashery#424 lands

**Pull request merged**